addons/website_livechat/models/discuss_channel.py

```diff
@@ -8,6 +8,10 @@
 class DiscussChannel(models.Model):
     _inherit = 'discuss.channel'
 
+    is_pending_chat_request = fields.Boolean(
+        "When created from an operator, whether the channel is yet to be opened on the visitor side.",
+        index="btree_not_null",
```

Comment:
`livechat_visitor_id` in the search seems pretty restrictive already, I don't think we need an extra index